src/sonic-config-engine/minigraph.py

```diff
@@ -1158,9 +1164,9 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     # if erspan_dst:
     #     lo_addr = '0.0.0.0'
     #     for lo in lo_intfs:
-    #         lo_network = ipaddress.IPNetwork(lo[1])
+    #         lo_network = ipaddress.ip_network(UNICODE_TYPE(lo[1]), False)
     #         if lo_network.version == 4:
-    #             lo_addr = str(lo_network.ip)
+    #             lo_addr = str(lo_network.network_address)
```

Comment:
nit: str->UNICODE_TYPE

Reply:
This is commented out. What about on line 666 above? It appears you also didn't change that from `str` to `UNICODE_TYPE`. Are you sure we want this as unicode?

Reply:
I get your point. Never mind the commented code, however a question regarding the clients if conversion might be required downstream???

Reply:
Not sure I understand your question, but I prefer not changing that in this PR. Once this is merged, if you determine it is necessary, please open a separate PR.